Update doc (#21)
uperl/NewFangle: README.md

always make sense to reproduce everything that is in the C-SDK documentation, so
it is recommended that you review it before getting started.

> This module requires a running `newrelic-daemon`, in other words the service `newrelic-infra` will return an initialization like this:

Reviewer: ```suggestion
This module requires a running `newrelic-daemon`, otherwise the service `newrelic-infra` will return an initialization diagnostic like this:
```

```
2021-05-27 06:41:27.160 +0000 (23284 23284) error: failed to connect to the daemon using a timeout of 0 ms at the path /tmp/.newrelic.sock
2021-05-27 06:41:27.160 +0000 (23284 23284) error: error initialising libnewrelic; cannot create application
```

I've called this module [NewFangle](https://metacpan.org/pod/NewFangle) in the hopes that one day NewRelic will write
native Perl bindings and they can use the more obvious NewRelic namespace.

# FUNCTIONS

These may be imported on request using [Exporter](https://metacpan.org/pod/Exporter).

For instance:
```perl

> use NewFangle qw/newrelic_init/;

Reviewer: ```suggestion
use NewFangle qw( newrelic_init );
```
style nit
